src/containers/nav/desktop/CurrentlyPlaying.module.css

```diff
@@ -32,6 +31,39 @@
   cursor: pointer;
 }
 
+.bottomRightContainer {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  width: 50px;
+  height: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.visualizerIconContainer {
+  padding: 5px;
+  background-color: white;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.visualizerIcon {
+  width: 22px;
+  height: 22px;
+}
+.visualizerIcon:hover {
+  transform: scale(1.10);
+}
+.visualizerIcon:hover path {
+  fill: #CC0FE0
```

Comment:
`fill: var(--primary)`

or perhaps `var(--static-primary)`

depending on whether we want this color to change with dark mode